src/mixxxapplication.cpp

```diff
@@ -17,27 +17,28 @@
 #include "util/fileinfo.h"
 #include "util/math.h"
 
-// When linking Qt statically on Windows we have to Q_IMPORT_PLUGIN all the
+// When linking Qt statically we have to Q_IMPORT_PLUGIN all the
 // plugins we link in build/depends.py.
```

Comment:
This comment is out of date.
```suggestion
// When linking Qt statically, the Q_IMPORT_PLUGIN is needed for each linked plugin.
// https://doc.qt.io/qt-5/plugins-howto.html#details-of-linking-static-plugins
```